.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       - checkout
       - restore_cache:
           key: tsickle-{{ checksum "yarn.lock" }}
-      - run: bazel run @yarn//:yarn
+      - run: yarn install
```

Comment:
add a comment why we don't use the yarn installed by bazel
(in general, any time you have to fix/workaround something, leave a comment explaining)